### ISSUE TITLE
Create a first Dashboard release with tagged docker images

### ISSUE THREAD

**Issue by mnuttall** [tektoncd/dashboard#363]
We'd like to have a release of dashboard to accompany the next Tekton release. Work with @dlorenc to understand what's needed so that we have a stable, released image in gcr.io that we can address in the same way as the pipeline release does, e.g. `gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/webhook@sha256:496e36b8723a668ac3531acc26512c123342da7827c10386b571aa975d6a47e7`

Document what is needed to perform a release, what the steps are, etc and update Development.md accordingly.

**Comment by a-roberts** [comment]
@dlorenc @vdemeester @bobcatfish this is something we'd love to be able to do, what should we do/try for our first dummy release please?

We'll need a fixed image that we can reference in our GitHub release, which we know how to do - do we literally just docker build and push with, say, a 1.0 tag on gcr through the Prow SA? 

How is this done for tektoncd/pipeline so we can follow along, are there any plumbing changes we should make?

**Comment by a-roberts** [comment]
Here's some juicy info from @vdemeester, who's mentioned we may need to fiddle with permissions and the like too. This is a good place to start:
https://github.com/tektoncd/pipeline/tree/master/tekton
https://github.com/tektoncd/pipeline/pull/1095

**Comment by dlorenc** [comment]
> We'll need a fixed image that we can reference in our GitHub release, which we know how to do - do we literally just docker build and push with, say, a 1.0 tag on gcr through the Prow SA?

Yep, prow *should* have all the permissions you need. If you run into issues there let me know.

**Comment by a-roberts** [comment]
Thanks Dan, so I've forked the repo and had a play (see https://github.com/a-roberts/pipeline/tree/freshmaster/tekton); replacing mentions of `pipeline` with `dashboard`. It's not clear what to do next though - I'd be happy with just pushing to my own Dockerhub, e.g. through something running on GKE from a Prow job, and then eventually gcr.io/tekton-releases I think.

There are lots of todos unfortunately, not sure where I should be doing the k8s commands too (really just on my Mac? Or pointing to a cluster on GKE? I'm a complete Google cloud novice!).

**Comment by a-roberts** [comment]
^^ notes added above that should prove useful, I'm "all over" this one but out Thursday and have a half day Friday, and this is a really hot item so I'm cool with anybody else picking it up or wanting to buddy up.

**Comment by a-roberts** [comment]
See https://github.com/tektoncd/dashboard/pull/408

@mnuttall and I worked with @vdemeester today to get a draft release of the Tekton Dashboard and webhooks extension created. I've added the release notes/attributions/feature list for both components and we're planning to include three assets.

The main dashboard yaml for all to use, one yaml for use with OpenShift only with HTTPS, and one for the extension. The first two yamls refer to the same image btw.
